src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -169,10 +169,12 @@ public void keyboardShortcutsTest() {
         // test shortcut on repo dropdown
         doubleClick(repoSelectorComboBox);
         pushKeys(SWITCH_DEFAULT_REPO);
-        waitUntilNodeAppears("#dummy/dummy_col1_1");
+        // wait for issue 9 to appear then click on it
+        // issue 9 is chosen instead of issue 10 due to a problem with finding issue 10's node
```

Comment:
Yes, at least mention that issue 10 cannot be found due to it being the first card in the panel.